### PR TITLE
Update documentation

### LINKED ISSUE #4: Update `CONTRIBUTING.md` file with how to contribute steps.

### PR DESCRIPTION
**Pull request by Abhirv08**
Solved issue #4 
Added the way to create a new issue. Also, added a way to start contributing to the project.

### CODE REVIEW

**Review by soyo-kaze — APPROVED** [review]
Great Work !! 🚀🧑‍🍳

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,10 +1,40 @@
...
+- Give a suitable title regarding the issue like "Update Readme.md", "Replace React context API with Redux", etc.
 
-## Pull Request Process
-- Ensure any install or build dependencies are removed before the end of the layer when doing a build.
-- Update the README.md with details of changes to the interface, this includes new environment variables, exposed ports, useful file locations and container parameters.
-- Increase the version numbers in any examples files and the README.md to the new version that this pull request would represent.
-- You may merge the Pull Request in once you have the sign-off of two other developers, or if you do not have permission to do that, you may request the second reviewer to merge it for you.
+- Describe your issue. If you think there is need of any image for better explanation, attach image too.
+
+- Then briefly explain what are going to do to solve the issue.
```
Comment (soyo-kaze) [inline CONTRIBUTING.md]: What "you" are going to do

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,10 +1,40 @@
...
+ &nbsp; ```git commit -m "your_commit_message"```
+- to push your changes to github. Use command
+<br/> &nbsp; ```git push -u origin {branch name}``` 
+
+## Raising Your 1st PR <br/>
+
+- After pushing your code, go to the forked repository. There you will see a section like this. <br />
+ <br><img src="./public/assets/pullreq.png"/>
+
+- click on compare and pull request. Such page will appear. Write meaningfull message and title which describes your changes (title should contain the issue number which you can see beside the title of issue)<br/>
```
Comment (soyo-kaze) [inline CONTRIBUTING.md]: - meaningfull --> meaningful

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,10 +1,40 @@
...
+<br/> <img src="./public/assets/PR.png"/>
+- click on create pull request. <br/>
```
Comment (soyo-kaze) [inline CONTRIBUTING.md]: All sentences should start with a capital letter.

**Review by soyo-kaze — CHANGES_REQUESTED** [review]
Just few typos and fixes.

**Comment by Abhirv08** [comment]
@soyo-kaze I have solved the typos.

**Comment by soyo-kaze** [comment]
I think you have accidentally closed the PR,😂 I am reopening it

File: CONTRIBUTING.md (hunk 3 above)
Comment (soyo-kaze) [reply CONTRIBUTING.md]: As you can see here all the points start with small letters just change them to capital letter. @Abhirv08

Comment (Abhirv08) [reply CONTRIBUTING.md]: sorry, I will fix it soon.😥

Comment (soyo-kaze) [reply CONTRIBUTING.md]: No worries

**Comment by soyo-kaze** [comment]
Great work @Abhirv08 !! 🚀🚀

**Review by soyo-kaze — APPROVED** [review]

**Pull request merged**